x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_alert_assignees_actions.tsx

```diff
@@ -50,6 +50,8 @@ export const useAlertAssigneesActions = ({
     ];
   }, [alertId, ecsRowData._index, ecsRowData?.kibana?.alert.workflow_assignee_ids]);
 
+  const currentAssignees = alertAssigneeData[0].data[0].value;
```

Comment:
You can add  `currentAssignees` definition before the `alertAssigneeData` and then re-use it. Something like:
```
  const currentAssignees = useMemo(
    () => ecsRowData?.kibana?.alert.workflow_assignee_ids ?? [],
    [ecsRowData?.kibana?.alert.workflow_assignee_ids]
  );
  const alertAssigneeData = useMemo(() => {
    return [
      {
        _id: alertId,
        _index: ecsRowData._index ?? '',
        data: [
          {
            field: ALERT_WORKFLOW_ASSIGNEE_IDS,
            value: currentAssignees,
          },
        ],
        ecs: {
          _id: alertId,
          _index: ecsRowData._index ?? '',
        },
      },
    ];
  }, [alertId, currentAssignees, ecsRowData._index]);
```